src/ui/map.ts

```diff
@@ -28,22 +30,49 @@ const defaultMarkerOptions: Partial<maplibregl.MarkerOptions> = {
   color: '#000257',
 };
 
+const uuidRegex = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/;
+
 const createStyleURL = (options: RadarOptions, style: string = DEFAULT_STYLE) => (
   `${options.host}/maps/styles/${style}?publishableKey=${options.publishableKey}`
 );
 
-// use formatted style URL if using one of Radar's out-of-the-box styles
+const getMarkerOptions = (marker: Marker): MarkerOptions => {
+  const markerOptions: MarkerOptions = {
+    // element: marker.getElement(),
+    color: marker._color,
+    scale: marker._scale,
+    offset: marker.getOffset(),
+    anchor: marker._anchor,
+    draggable: marker.isDraggable(),
+    clickTolerance: marker._clickTolerance,
+    rotation: marker.getRotation(),
+    rotationAlignment: marker.getRotationAlignment(),
+    pitchAlignment: marker.getPitchAlignment()
+  }
+
+  return markerOptions;
+}
+
+/** Check if style is a Radar style or a custom style */
+const isRadarStyle = (style: string) => {
+  return RADAR_STYLES.includes(style) || uuidRegex.test(style)
```

Comment:
I'm not so sure we need to validate the UUID here. We should maybe just consider any style to be a Radar style.

The other thing we could do, is prefix the UUIDs with `radar://`. Probably a larger discussion.